tests/ExampleTest.php

```diff
@@ -20,8 +23,17 @@ class ExampleTest extends TestCase {
   use MinkSetup;
   use DrupalSetup;
   use NodeCreationTrait;
+  use UserCreationTrait;
 
-  /**
+  // The entity creation traits need this.
+    use RandomGeneratorTrait;
```

Comment:
this belongs with the entity creation traits then, no?

Reply:
It can't. As soon as more than one is in use (if they each declare their own use of that trait), a fatal error is thrown:

```
Fatal error: Trait method randomString has not been applied, because there are collisions with other trait methods on ExampleTest
```